### PR TITLE
build: remove downloading of credentials

### PR DESCRIPTION
**Pull request by mediumTaj**
This pull request removes downloading of internal watson service credentials in the core build process.

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ environment:
         secure: +B2bs86RVtJtlbkB+cTf9bkqnNlFJi/PbBBPzR5jlUlLLZoOc+ZgqgQLwee4tCT+ #pragma: whitelist secret
 install:
 # Get the latest stable version of Node.js or io.js
-- ps: Install-Product node $env:nodejs_version
+- ps: Install-Product node 12.13.0
```
Comment (nan2iz) [inline appveyor.yml]: do you want to update `nodejs_version` variable instead of put the actual version here?
But I'm okay with this too.

Comment (kevinkowa) [reply appveyor.yml]: yes, I'm wondering if we were setting this on appveyor

Comment (mediumTaj) [reply appveyor.yml]: ah sorry i missed this. Yeah that would be good - where do we define nodejs_version?

Comment (kevinkowa) [reply appveyor.yml]: I found this https://github.com/appveyor/ci/issues/2921. I think it defaults to the latest version, but we can set it on the yaml file.

**Review by kevinkowa — APPROVED** [review]
lgtm!

**Review by nan2iz — APPROVED** [review]

File: appveyor.yml [hunk 2]
```diff
@@ -17,6 +17,8 @@ environment:
         secure: gevsMhy8RTWMdf7MjOnIo5QaN6JpL9DPK6I+Go5ByZir5LDwyFsv9hO6nBjGTg8n #pragma: whitelist secret
     GITHUB_TOKEN:
         secure: +B2bs86RVtJtlbkB+cTf9bkqnNlFJi/PbBBPzR5jlUlLLZoOc+ZgqgQLwee4tCT+ #pragma: whitelist secret
+    nodejs_version: 12.13.0
```
Comment (nan2iz) [inline appveyor.yml:20]: oh sorry @mediumTaj 

I just noticed this. You may want to add additional independent before `nodejs_version`. The same level with the `secure`

**Review by nan2iz — APPROVED** [review]

**Pull request merged**